doc/contributing/distribution.md

```diff
@@ -0,0 +1,24 @@
+# Node.js Distribution Policy
+
+This document describes some policies around what is and is not included in the
+Node.js distribution.
+
+## Inclusion
+
+Node.js includes some external projects that the Node.js team does not maintain.
```

Comment:
I’m referring to the Node.js distribution, not the repo. Should this say “The Node.js distribution”? I assumed it was obvious because the entire document is about the Node.js distribution.

Reply:
Maybe say "it" if you don't want to repeat it
```suggestion
It includes some external projects that the Node.js project does not maintain.
```

Reply:
I don’t think we should start a section with “It.” Is that what you’re looking for?

```suggestion
The Node.js distribution includes some external projects that the Node.js project does not maintain.
```

Reply:
That would work yes. Another idea would be to use "Official distributions", no string feelings though

Reply:
Saying “official distributions” implies that maybe there are other policies for “unofficial” distributions, but I don’t want to open up that can of worms.